iroha_version/iroha_version_derive/tests/ui/fail_invalid_range_equal.stderr

```diff
@@ -0,0 +1,15 @@
+error: The end version should be higher then the start version.
```

Comment:
I guess it shouldn't be committed.

Reply:
Or if it validates compiler output it should be ok

Reply:
Yes, it's for error message validation.